lib/IRGen/IRGen.cpp

```diff
@@ -269,6 +270,7 @@ class MD5Stream : public llvm::raw_ostream {
     Hash.final(Result);
   }
 };
+}
```

Comment:
Ah, yeah, I didn't think about it because it's such a short class, but you're right.